api/src/app.py

```diff
@@ -191,3 +193,18 @@ def robots() -> str:
         Allow: /docs
         Disallow: /
         """
+
+
+def register_well_known(app: APIFlask) -> None:
+    @app.route("/.well-known/pki-validation/<file_name>")
+    @app.doc(hide=True)
+    def get_dv_verification_content(file_name: str) -> tuple:
+        try:
+            with open(
+                f"{Path(__file__).resolve().parent}/static/domain_verification_assets/{file_name}"
+            ) as f:
+                return f.read(), 200
+        except Exception as e:
+            message = f"Could not read {file_name} dv contents: {e}"
+            logger.warning(message)
```

Comment:
I wouldn't make this a warning. Anything above info we treat as something to alert us - 4xx scenarios aren't a problem for us (eg. we wouldn't want to log a warning/error if a user hits the GET /opportunity endpoint with an ID that doesn't exist).